packages/runtime/src/prop-controllers/descriptors.ts

```diff
@@ -47,7 +47,7 @@ export type IndexSignatureHack<T> = T extends Record<string, any>
   ? { [K in keyof T]: IndexSignatureHack<T[K]> }
   : T
 
-export type Device = 'desktop' | 'tablet' | 'mobile'
+export type Device = 'desktop' | 'tablet' | 'mobile' | string
```

Comment:
I see we're doing this all over the place, I'm guessing the type will just be `string` eventually?